README.md

```diff
@@ -14,6 +14,20 @@
 
 # data load tool (dlt)
 
+```python
+import dlt
+from chess import chess
```

Comment:
I'd consider adding a brief comment explaining the purpose of the `chess` import for those who are not familiar with it. Although there's a demo of its usage later on, it may not be immediately obvious that this is our custom code. Something like: 'chess is a utility function to fetch chess game data from the chess.com API'.